common/scala/src/main/scala/whisk/http/BasicHttpService.scala

```diff
@@ -90,7 +91,13 @@ trait BasicHttpService extends Directives with TransactionCounter {
   }
 
   /** Assigns transaction id to every request. */
-  protected val assignId = extract(_ => transid())
+  protected def assignId = HeaderDirectives.optionalHeaderValueByName(OW_EXTRA_LOGGING_HEADER) flatMap { headerValue =>
```

Comment:
add a comment? e.g. `extract headers from HTTP request that indicates if additional logging should be enabled for this transaction. Allowed values are ...`

Reply:
I have added a comment explaining the header's purpose

Reply:
This header will be visible to web actions - should we strip it?

Reply:
Good catch. The header will be removed.